docs/javascript/v2/guides/deployment-iframe-integration.mdx

```diff
@@ -0,0 +1,95 @@
+
+#### Deploying via Git (Recommended)
+
+If you had forked 100ms-web and have customized it, it is highly likely you are using git. If you did, You can directly deploy to Netlify. Netlify supports github, gitlabs and bitbucket. Netlify has great documentation on how to do it.
+
+-   [A Step-by-Step Guide: Deploying on Netlify
+    ](https://www.netlify.com/blog/2016/09/29/a-step-by-step-guide-deploying-on-netlify/)
+-   [Netlify Tutorial - Deploying from Git(Video)](https://youtu.be/4h8B080Mv4U)
+
+#### Deploying the Files Directly
+
+If you want to directly upload your files, Netlify provides a very easy drag & drop interface for it.
+
+-   [Netlify Tutorial - Drag and drop deploys on Netlify
+    (Video)](https://youtu.be/etZ9HSUoTPU)
+
+> We recommend using Git in general if you are planning to use this code for production. Only use the direct method if you want to quickly test something or do some proof-of-concept.
+
+If you are interested in Netlify's CI/CD or just want to learn more about deploying to Netlify, please check their docs [here](https://docs.netlify.com/site-deploys/create-deploys/)
+
+### Vercel
+
+Deploying to Vercel is pretty much straightforward similar to Netlify. You can start with a [New Vercel app](https://vercel.com/new).
+In that page, import your repository from git, Click next
+
+![vercel_wiki_0](/docs/v2/vercel_0.png)
+
+In the next page, create a team first
+
+![vercel_wiki_1](/docs/v2/vercel_1.png)
+
+Then in **Configure Project** section, select **Framework preset** as **Create React App** and then add any extra build settings you want to add. Make sure to add the environment variables for the app below that. For more details See [Environment Variables](https://github.com/100mslive/100ms-web/wiki/Environment-Variables)
+
+![vercel_wiki_2](/docs/v2/vercel_2.png)
+
+> You can change the environment variables any time later in the settings page of your project dashboard. Every time you change a variable, add/remove a variable, you have to rebuild and make a new deployment for the environment variables to take effect. This is a mandatory step and must be done.
+
+Now click deploy. Wait for the project to be deployed. Once it's successful, you should see a screen like this
+
+Click **Go to dashboard**
+
+![vercel_wiki_3](/docs/v2/vercel_3.png)
+
+In the dashboard page, you should be able to see the app by clicking "Visit" button on the deployment screen. Click it to see your app in action.
+
+That's it. You are all done.
+
+## Iframe Integration
+
+If you want a quick integration of a fully featured video conferencing app inside your application within a day, iframe is the way to go.
+
+As discussed above, you could fork and deploy our [sample web app](https://github.com/100mslive/100ms-web/) and use that URL in an iframe or use one of 100ms' links from the [rooms page](https://dashboard.100ms.live/rooms) in your dashboard to make it even quicker.
+
+```html
+<iframe
```

Comment:
It'd be helpful if we give an example for src. 
Also, an example of customizing the UI and allowed values for "allow" would help.